Zoo/Zoo.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Runtime.CompilerServices;
+using System.Security.AccessControl;
+using System.Text;
+using System.Threading.Tasks;
+using Zoo.Animals;
+using Zoo.Animals.AirAnimals;
+using Zoo.Animals.LandAnimals;
+using Zoo.Animals.MixedAnimals;
+using Zoo.Animals.SeaAnimals;
+using Zoo.Workers;
+
+namespace Zoo
+{
+    public class Zoo
+    {
+        public static AreaType LandTour = new AreaType(Area.Land);
+        public static AreaType AirTour = new AreaType(Area.Air);
+        public static AreaType SeaTour = new AreaType(Area.Sea);
+        public static AreaType MixTour = new AreaType(Area.Mix);
+
+        public List<Animal> Animals;
+        public List<Worker> Workers;
+        public List<Person> Visitors;
+
+        public const int SECONDS_IN_DAY = 120;
+
+        public delegate void FieldChangedEventHandler();
+        public event FieldChangedEventHandler CurrentTimeChanged;
+
+        public int CurrentTime = 0; //seconds
+
+        public Zoo()
+        {
+            this.ZooInit();
+        }
+
+        protected virtual void OnCurrentTimeChange()
+        {
+            //runs the methods that need to happen as every second passes. (ex: check if worker's time to work).
+            CurrentTimeChanged?.Invoke();
+        }
+
+        //make the worker that should work at this time work.
+        public void WorkWokers()
+        {
+            foreach (Worker worker in this.Workers)
+            {
+                if (worker.IsWorkingAtTime(CurrentTime)) {
+                    ZooManagerSingleton.Instance.AddDocumantation($"{worker.Name} doing work");
+                    worker.DoWork();
+                }
+            }
+        }
+
+        public void ZooInit() {
+            CurrentTimeChanged += new FieldChangedEventHandler(WorkWokers);
+            Animal animalAir1 = new Eagle("Moshe");
```

Comment:
I'll pass on this because D SHIT BLACK AF, but only for protocol- no numbers in variables' names.